apps/condo/pages/ticket/index.tsx

```diff
@@ -58,7 +56,18 @@ const GET_ALL_PROPERTIES_QUERY = gql`
     }
 `
 
-async function _search (client, query, variables) {
+const GET_ALL_ORGANIZATION_EMPLOYEE_QUERY = gql`
```

Comment:
we should put all GQL queries inside `gql.js`. Or in this case, is better to use `OrganizationEmployee.useObjects()` hook